### ISSUE TITLE
How to register and login with this package?

### ISSUE THREAD

**Issue by zwl1619** [Folkloreatelier/laravel-graphql#263]
front end: apollo + relay+ react native.
back end: laravel + laravel-graphql

How to register and login? The out-of-box `auth` component could be used here?

**Comment by kevinvdburgt** [comment]
The Laravel out-of-box `auth` works fine, however, I wouldn't recommend using session for it. Maybe take a loot at JWT tokens for example.

Or you can create your own autorization token based mechanic :)

**Comment by codeitlikemiley** [comment]
yes JWT token is enough, but if you like laravel passport it is still possible but, i doubt its a perfect use case to use passport ...

**Comment by SonarSoftware** [comment]
I use Passport, it works perfectly.

**Comment by kikoseijo** [comment]
Full example using passport:
https://github.com/kikoseijo/react-relay-app

**Comment by zwl1619** [comment]
@kikoseijo That is cool!
